### ISSUE TITLE
DockerClientTest.java

### ISSUE THREAD

**Issue by jrmillerwork** [docker-java/docker-java#492]
The loop: `for (String command : commands) {
            LOG.info("Running command: [{}]", command);

            CreateContainerResponse container = dockerClient.createContainerCmd("busybox").withCmd(commands).exec();
            dockerClient.startContainerCmd(container.getId());

            int exitcode = dockerClient.waitContainerCmd(container.getId()).exec(new WaitContainerResultCallback())
                    .awaitStatusCode();
            assertThat(exitcode, equalTo(0));
        }
`
Seems to be executing commands rather than command -- is this correct?

**Comment by marcuslinke** [comment]
@jrmillerwork You're right. Probably a refactoring mistake. Initial test is here:https://github.com/kpelykh/docker-java/blob/4be3f66c00d98d8325ec29a840d8e94014d239ec/src/test/java/com/kpelykh/docker/client/test/DockerClientTest.java#L530

There are many simmilar tests [now](https://github.com/docker-java/docker-java/blob/master/src/test/java/com/github/dockerjava/core/command/StartContainerCmdImplTest.java) so this one could be removed I think.

**Comment by KostyaSha** [comment]
Wow! It used created object as request! 

@marcuslinke i think it mostly needed to have some *Test for DockerClient class. Fixed in #499
